setup.py

```diff
@@ -46,6 +46,12 @@
             if "=" in dep
         ],
     },
+    entry_points={
+        "console_scripts": [
+            "fixit-run=fixit.cli::run_rules",
+            "fixit-apply=fixit.cli::apply_fix",
```

Comment:
Thanks for contributing.
This should be just one single colon.

Please provide a test plan and result in PR summary. 
You can do `python setup.py sdist` and then use pipx to install the built artifact to verify whether it works.

Please also sign the CLA.